app.cpp

```diff
@@ -421,7 +421,7 @@ void App::Run()
 }
 
 #ifdef USE_PAM
-bool App::AuthenticateUser(bool focuspass)
+bool App::AuthenticateUser(bool focuspass, bool hrsauth)
```

Comment:
You need to add the `, bool hrsauth` passing to the non-PAM version of the function too at #448